samples/nrf9160/nrf_cloud_mqtt_multi_service/src/fota_support.c

```diff
@@ -13,10 +13,13 @@
 
 LOG_MODULE_REGISTER(fota_support, CONFIG_MQTT_MULTI_SERVICE_LOG_LEVEL);
 
-#if defined(CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE)
+#if (IS_ENABLED(CONFIG_NRF_CLOUD_FOTA_FULL_MODEM_UPDATE) && \
+	!IS_ENABLED(CONFIG_DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION))
 /* Full modem FOTA requires external flash to hold the full modem image.
  * Below is the external flash device present on the nRF9160 DK version
  * 0.14.0 and higher.
+ * If an external partition is used, it is not necessary to specify the
+ * flash device.
```

Comment:
I think since the overlay enables the external partition, it wpuld be fine to remove `FULL_MODEM_FOTA_FLASH_DEVICE` altogether, like what you've done with one of the other samples

Reply:
yeah, i mostly left this here just as an example...  maybe it is unecessary and/or confusing?

Reply:
Yeaaah, I think it's just barely beyond the scope of being demonstrated in MMSS. I agree there is value in demonstrating explicit device selection somewhere, but in this location I personally feel it is unnecessary